apps/mobile/src/queries/balance/sip10-balance.query.ts

```diff
@@ -13,6 +13,9 @@ export function useSip10TotalBalance() {
   return toFetchState(useSip10AggregateBalanceQuery(addresses));
 }
 
+/**
```

Comment:
Lets just get rid of this if not needed. We only need `useSip10TotalBalance` and `useSip10AccountBalance`